src/components/Navbar/style.css

```diff
@@ -0,0 +1,45 @@
+.Bookstore-CMS {
+  width: 240px;
+  height: 37px;
+  margin: 6px 47px 2px 0;
+  font-size: 30px;
+  font-weight: 700;
+  letter-spacing: normal;
+  color: #0290ff;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+}
+
+.panel-bg {
+  width: 1400px;
+  height: 100px;
+  margin: 0 0 20px;
+  padding: 23px 99px 27px 100px;
+  background-color: #fff;
+}
+
+.BOOKS {
+  width: 59px;
+  margin: 0 41px 11px 0;
+}
+
+.CATEGORIES {
+  width: 105px;
+  opacity: 0.5;
+}
+
+.Oval {
+  padding: 14px;
+  border: 1px solid #e8e8e8;
+  border-radius: 50%;
+}
+
+.Mask {
+  width: 17px;
+  height: 17px;
+  color: #0290ff;
+  font-size: 17px;
+  display: block;
+  margin: auto;
+}
```

Comment:
- [ ] <b>REQUIRED:</b> There are still few things we need to fix to make our app look more like the provided design. Kindly review the following issues and make sure to fix them.

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_10_8_2021_4-57-51 PM](https://user-images.githubusercontent.com/45217364/136579256-5a190102-08f8-4c7b-a3ed-1b3b8962ceb7.png) |
   - As mentioned in the previous review, The app should not be fullscreen. It should be like a card in the center of the page as it is in the mockup design.
   - As mentioned in the previous review, The categories selector should be in the toolbar as it is in the mockup design
   -And There are should be a line separating the booklist and form